pymdb.py

```diff
@@ -5,93 +5,85 @@
 
 
 def top_250():
-    """Pulls out movie names from the Imdb Top 250 webpage and adds them to a dictionary with its position.
+    """Pulls out movie names from the Imdb Top 250 webpage and adds them to a
+    dictionary with its position.
     Returns dictionary"""
 
-    movielist=dict()
-    i=1
-    url='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url).read()
-    link = re.findall('title=".*?dir.*?>(.*?)</a>',file)
+    movielist = dict()
+    i = 1
+    url = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url).read()
+    link = re.findall('title=".*?dir.*?>(.*?)</a>', file)
     for links in link:
-        
-        movielist[i]=links
-        i+=1
+
+        movielist[i] = links
+        i += 1
     return movielist
 
+
 def top250_yearcount():
-    """Creates a dictionary of number of movies of a particular year in the Imdb Top 250.
+    """Creates a dictionary of number of movies of a particular year in the
+    Imdb Top 250.
     Returns a dictionary"""
 
-    i=1
-    yearcount=dict()
-    
-    url='http://www.imdb.com/chart/top'
-    file= urllib.urlopen(url).read()
+    i = 1
+    yearcount = dict()
 
-            
-         
-    years = re.findall('secondaryInfo">\((.*?)\)</span>',file)
+    url = 'http://www.imdb.com/chart/top'
+    file = urllib.urlopen(url).read()

```

Comment:
This warning comes up using PyCharm: `Shadows built-in name 'file'`

This variable name should be changed.